Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,5 +1,63 @@
-function setAlarm() {}
+const heading = document.querySelector("#timeRemaining");
+const stopBtn = document.querySelector("#stop");
+heading.innerText = "Time Remaining: 00:00";
+const headerText = 'Remaining Time:'
```

Comment:
This code now exhibits that bug I suggested could be possible - the header text changes from "Time Remaining" to "Remaining Time" when you start the timer. This feels like a bug?

How could you structure this code to make that bug less likely to happen?

Reply:
I think I can play with the ```heading.innerText = "Time Remaining: 00:00"```. to extract the ```Time Remaining:``` and store it into a variable. I am not 100% sure but that's the idea I have at the moment

Reply:
You have already written a function that sets the heading properly given a number of seconds - could you think how you could re-use that function, rather than writing `heading.innerText = ...` again here?

Reply:
Thinking about it now

Reply:
I can call ```formatAndDisplayTime``` with ```totalTimeInSeconds``` set to 0 as the default value.